app/src/main/java/com/google/droidjump/MainActivity.java

```diff
@@ -24,9 +24,27 @@
  */
 public class MainActivity extends AppCompatActivity {
 
+    private int levelsCount;
+    private int currentLevel;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        levelsCount = GameConstants.GAME_LEVELS_COUNT;
+        currentLevel = 5;
```

Comment:
Why "5"? Shouldn't we start with level "1" or "0"?

Reply:
it was only for testing purposes to show the levels the screen would look. In the next PR, we store the current level in SharedPreferences. Should I change it to "1"?

Reply:
Yes, please change it to "0" or "1" or remove it entirely.